.serena/memories/suggested_commands.md

```
@@ -0,0 +1,186 @@
+- テスト実行結果のレポートを表示
+
+## デプロイ
+
+### Cloudflare Workersへデプロイ
+```bash
+pnpm deploy
+```
+- ビルド + デプロイを一括実行
+- `./dist/` の内容をCloudflare Workersにデプロイ
+- ワーカー名: `me`
+- デプロイ先: https://ta93abe.com
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)